tests/test_merger.py

```diff
@@ -17,13 +17,13 @@
 sys.path.append(PROJECT_ROOT)
 
 
-def test_merge():
+def merger_operate(merger):
```

Comment:
I would prefer if you leave this test as it is and make something like this:

```suggestion
def merger_operate(merger):
    pdf_path = os.path.join(RESOURCE_ROOT, "crazyones.pdf")
    outline = os.path.join(RESOURCE_ROOT, "pdflatex-outline.pdf")
    merger.append(pdf_path)
    merger.append(outline)

def test_merge():
```